client/sites-dashboard/components/sites-grid-item.tsx

```diff
@@ -201,13 +205,26 @@ export const SitesGridItem = memo( ( props: SitesGridItemProps ) => {
 						<Truncated>{ displaySiteUrl( siteUrl ) }</Truncated>
 					</SiteUrl>
 					<WithAtomicTransfer site={ site }>
-						{ ( result ) =>
-							result.wasTransferring ? (
-								<TransferNoticeWrapper { ...result } />
-							) : (
-								<>{ showLaunchNag && <SiteLaunchNag site={ site } /> }</>
-							)
-						}
+						{ ( result ) => {
+							if ( result.wasTransferring ) {
+								return <TransferNoticeWrapper { ...result } />;
+							} else if ( showLaunchNag && 'unlaunched' === site.launch_status ) {
+								return <SiteLaunchNag site={ site } />;
+							} else if ( site.is_wpcom_atomic ) {
+								return (
+									<a
+										href={ wpAdminUrl }
+										title={ __( 'Visit Wordpress Admin Dashboard' ) }
```

Comment:
Small nit: `P` in WordPress should be capitalized :). (My translator called it out ;) )